### ISSUE TITLE
Global 1 degree setup; output variable names from different setups

### ISSUE THREAD

**Issue by Sougata18** [team-ocean/veros#378]
Hello everyone! I have been trying to run one of the basic setups available in Versos, i.e. , the gloabl_1deg setup on my PC. But each time I try to run, its showing a "killed" message in the terminal. My PC has 8 GB RAM ; Is it too low to run the setup on my PC?

Also I would like to know what the output variable names actually mean. There are some variables for example the "bolus_depth" and "bolus_iso" from the global_4deg setup which inspite of having the same coordinates and attributes (i.e., meridional transport), represent different things. I looked into the official documentation but it has description for only model variables. 
Thanks for any kind of help!

**Comment by dionhaefner** [comment]
> My PC has 8 GB RAM ; Is it too low to run the setup on my PC?

Yes, I would recommend at least 12 GB of memory. You might be able to get by with less by using JAX and picking a different equation of state. Otherwise you can use `global_flexible` and set it to a resolution that is slightly above 1 degree.

> Also I would like to know what the output variable names actually mean. There are some variables for example the "bolus_depth" and "bolus_iso" from the global_4deg setup which inspite of having the same coordinates and attributes (i.e., meridional transport), represent different things. I looked into the official documentation but it has description for only model variables. Thanks for any kind of help!

You are right that the overturning variables are currently undocumented. To be honest I haven't looked at most of the outputs myself (just ported them from PyOM and ensured that the output matches) except the vertical streamfunction.

It looks like `bolus_iso` is the bolus (eddy-induced) meridional transport in isopycnal coordinates and `bolus_depth` that in z-coordinates. But I don't know the details, for example why you need both and how exactly they differ. If someone who knows more about this can contribute a better explanation I'd be grateful :)

**Comment by Sougata18** [comment]
Thanks a lot for the info!
